### ISSUE TITLE
Looking for volunteers

### ISSUE THREAD

**Issue by PalmerAL** [minbrowser/min#1080]
The volume of issues on this repository has grown large enough that I no longer have time to respond to them all, or to investigate all the issues that are being reported. If you're interested in helping out, let me know and I'll add you to the organization.

**Comment by KagamineP** [comment]
Hello, @PalmerAL. I interested about this and I can help you with that.

**Comment by PalmerAL** [comment]
@KagamineP Great, thank you! I've sent an invitation. Let me know if you have any questions or want to talk about it.

If anyone else is interested, please let me know!

**Comment by KagamineP** [comment]
> Let me know if you have any questions or want to talk about it.

Okay.

**Comment by Sestowner** [comment]
Hi, @PalmerAL. I'm interested in helping you.

**Comment by PalmerAL** [comment]
@Sestowner Thank you! Also let me know if you have any questions.

**Comment by ebrensi** [comment]
I'd love to but I'm a painfully slow coder and I am working on developing an open-source web app right now (https://www.heatflask.com).   I am willing to tackle a bug or two in Min, especially with regards to my app working on it.

**Comment by PalmerAL** [comment]
@ebrensi No problem, anything is appreciated.

**Comment by antfu** [comment]
I can help a bit as well :) Love to see this awesome project sustains.

**Comment by PalmerAL** [comment]
@antfu Great, thank you! I've added you to the organization so you can edit issues if you want. If you have any questions, let me know.

**Comment by antfu** [comment]
@PalmerAL Just to make sure the issue managing philosophy aligns with your preference before I work on.

### Issues Closing

To me, I believe managing a small number of issues will help the overall maintenance efficient and make new issues being better handled. I would advocate closing issues as much as we could. Is that good for you?

### Duplicated

Redirect to existing/older one and close the new one.

### Feature Requests

After quick browsing, I found a lot of fundamental features requests like supporting favicons in tabs. The initial ones might be raised years ago. For those you commented saying not going to support, would you like me to keep the discussion or just close them and labeled with `won't fix`?

---

Please let me know what you think or if you have any additional notes. After that, I can start reading the issues and labeling them when I got some time.

Thanks :)

**Comment by ebrensi** [comment]
Yes.  The issue count is overwhelming.  Please close as many as possible
and close any PRs that are not mergeable, and if the author feels like
updating it they can reopen it.

**Comment by PalmerAL** [comment]
Yeah, I agree that reducing the number of open issues would be good. Regarding the different categories:

* What you suggested for duplicates sounds fine.
* Questions - it's probably fine to close these if the question has been answered.
* Bugs - a few of these are random issues that someone had a few years ago, and that no one else has ever been able to reproduce; it's probably fine to close those. But there's also a lot of legitimate issues that have never been fixed, and that would be better to leave open. If you're not sure, I can look at it and try to figure it out.
* For feature requests that I decided not to do , closing with wontfix sounds good.
  * There's a lot of issues open for low-priority feature requests that I'm probably not going to work on, but that I would be fine with adding if somebody wanted to contribute it. It's probably fine to just leave those as-is, although I guess some kind of issue prioritization system could work as well.

Thanks for working on this!

@ebrensi I agree on the PRs, I'll go ahead and do that.

**Comment by antfu** [comment]
@PalmerAL I am reading issues from the very beginning and sorting them out to a FAQ in #1128. I would like to label them alongside, can I have the permission to create and modify labels? My idea would be something like

- OS: `os/mac` `os/win` `os/linux`
- Feature Request: `feat/ui` `feat/ux` `feat/shortcuts` `feat/web-standard` `feat/preference` etc
- Bugs: `bug` `bug/ui` `bug/security` `bug/electron` etc
- Status: `help wanted` `contribution welcome` `reproduction needed` `request for comments` etc

**Comment by PalmerAL** [comment]
It's looking a lot better, thanks for all the work you've done so far! (and thank you Sergiy and Dmitry as well!)

I'm definitely fine with you changing the labels! I think you have permission already, and I don't see any other permission levels listed in the Github UI for that. I guess if Github won't allow you to I can just create them myself (and you definitely should be able to add them to issues).

> OS: os/mac os/win os/linux

Sounds good; I think these can just be a rename of the labels that already exist.

> Feature Request: feat/ui feat/ux feat/shortcuts feat/web-standard feat/preference etc

I suspect the specific categories might need to be different - for example, web standards are going to be Chromium/Electron issues that we generally can't do anything about - but the general idea is fine.

> Bugs: bug bug/ui bug/security bug/electron etc

Security issues shouldn't be reported publicly, so it'd be better not to have a label for that. (Which reminds me, I need to write a document explaining how to report security issues, probably as part of a larger contributing guide. I'll make a PR with a draft soon).

> Status: help wanted contribution welcome reproduction needed request for comments etc

Sounds great!

**Comment by antfu** [comment]
@PalmerAL So far I can close issues or labeling them, but it still seems that I can not create/rename labels.

**Comment by PalmerAL** [comment]
I can't find any way to grant permission for that, so I've tried to update the labels to match what you suggested; let me know if there's any other ones you want me to create.

**Comment by Syndamia** [comment]
Hello @PalmerAL, I would like to help out. Although I don't have much JavaScript and Electron experience (I will spend some time learning them), I can atleast help out with the organisation of issues and fixing the obvious/simple ones.

**Comment by PalmerAL** [comment]
@Syndamia Sounds good, I appreciate it!

**Comment by kodumbeats** [comment]
HI @PalmerAL, I would like to help out as well. I've been using Min for a bit now, and I'd like to contribute to the project. Are you still looking for volunteers?

**Comment by PalmerAL** [comment]
@kodumbeats Great! I think most of the work of organizing the existing issues has been done, but there's always more being filed, so you're certainly welcome to help with that :)

**Comment by fullarray** [comment]
Let me know. I can help tackle a couple of open issues. As antfu said, there is a growing number of duplicate questions.

**Comment by PalmerAL** [comment]
@fullarray Thanks for the help, I've added you to the organization.

**Comment by Milad-Laly** [comment]
Hi @PalmerAL , I don't mind helping Min if you have any spare volunteer positions? I just want to contribute to the community and help Min develop faster and give something back to the community.

**Comment by PalmerAL** [comment]
@Milad-Laly Great, I've added you as well.

**Comment by Milo123459** [comment]
@PalmerAL I'd be happy to help :)

**Comment by urdadx** [comment]
Is this offer still opened? @PalmerAL

I would like to contribute to this project, I don't have much experience contributing to open source or writing Electron/JavaScript code but I've been using Min for quite sometime now and would like to see it grow ❤️.
I can start small by labelling issues or helping in any way I can..
